dhis-2/dhis-api/src/main/java/org/hisp/dhis/program/SingleEvent.java

```diff
@@ -162,7 +162,7 @@ public Date getScheduledDate() {
   }
 
   @Override
-  @JsonProperty("eventDate")
```

Comment:
mhm, so if this does not fail any test I wonder why we even have the Json annotations on these classes? we have our view models which we use to serialize to json. Can we remove all of these annotations here? or are these classes still put into json somewhere like a job/queue?

Reply:
I think that we can actually remove the annotations.
I will try it in the next PR when I rename `Event` to `TrackerEvent`